src/wavm.cpp

```diff
@@ -193,9 +227,49 @@ ExecutionResult WavmEngine::execute(
   evmc_message const& msg,
   bool meterInterfaceGas
 ) {
+
+  // clear stuff from previous run
+  wavm_host_module::exceptionIdx = wavm_host_module::HeraExceptionsEnum::None;
+  wavm_host_module::exceptionString = "";
+  // hope and prayer
+  Runtime::collectGarbage();
```

Comment:
why do we collect garbage redundantly here? (other call at L241)

Reply:
Yes, this is a "hope and prayer" that it will clean up something. Removing this is OK if you insist.